### PR TITLE
[chore] enable AWS cloud provider tests

### PR DESCRIPTION
**Pull request by shaneutt**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Enables some cloud provider tests which use to be disabled

https://jira.d2iq.com/browse/D2IQ-65347

### CODE REVIEW

**Comment by shaneutt** [comment]
Blocked on https://github.com/mesosphere/kubeaddons/pull/777

**Comment by makkes** [comment]
@shaneutt btw, we have the PR template for a reason. 😉

File: test/README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ In this directory you will find [go tests](https://golang.org/pkg/testing/) whic
 
 This uses the [Kubeaddons](https://github.com/mesosphere/kubeaddons) addon testing framework, which is [documented here](https://github.com/mesosphere/kubeaddons/blob/master/docs/test/framework.md).
 
+## Developer Notes
+
+When working on the tests, you may find that some symbols in the kubeaddons package appear to be unresolvable. This is due to the use of experimental features in the kubeaddons library. To avoid these problems in `vim` and other editors, make sure your local environment is set with the `experimental` build tag:
```
Comment (makkes) [inline test/README.md]: 👍

File: test/addons_test.go [hunk 2]
```diff
@@ -192,19 +205,37 @@ func testgroup(t *testing.T, groupname string) error {
 		}
 	}()
 
-	cluster, err := kind.NewCluster(version, cluster.CreateWithV1Alpha3Config(&v1alpha3.Cluster{
-		Nodes: []v1alpha3.Node{node},
-	}))
+	var tcluster test.Cluster
+	if groupname == "aws" || groupname == "azure" || groupname == "gce" {
```
Comment (makkes) [inline test/addons_test.go]:
```suggestion
	if groupname == "aws" || groupname == "azure" || groupname == "gcp" {
```

File: test/scripts/setup-konvoy.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+KONVOY_VERSION="v1.4.0"
```
Comment (makkes) [inline test/scripts/setup-konvoy.sh]: Shouldn't we test v1.4.1 by now?

File: test/scripts/setup-konvoy.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (makkes) [inline test/scripts/setup-konvoy.sh]: Where is this script used? I think it'd be helpful to have a comment here.

File: test/scripts/setup-konvoy.sh (hunk 3 above)
Comment (shaneutt) [reply test/scripts/setup-konvoy.sh]: That one doesn't appear to be published yet, but we can change this in CI without having to change the code when we need to.

File: test/scripts/setup-konvoy.sh (hunk 4 above)
Comment (shaneutt) [reply test/scripts/setup-konvoy.sh]:
```suggestion
#!/bin/bash
# NOTE: used by teamcity
```

File: test/groups.yaml [hunk 5]
```diff
@@ -94,6 +94,27 @@ localvolumeprovisioner:
...
     - "localvolumeprovisioner"
 
+# ------------------------------------------------------------------------------
+# AWS
+#
+# All AWS utility addons should be tested as a part of this group
+# ------------------------------------------------------------------------------
+aws:
+    - "defaultstorageclass-protection"
+    - "cert-manager"
```
Comment (dkoshkin) [inline test/groups.yaml]: Is `cert-manager` required for `awsebscsiprovisioner` ?

File: test/scripts/setup-konvoy.sh [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# NOTE: used by teamcity
+
+KONVOY_VERSION="v1.4.0"
```
Comment (dkoshkin) [inline test/scripts/setup-konvoy.sh]: Lets use `v1.4.1` as it contains a few bug fixes.

File: test/scripts/setup-konvoy.sh (hunk 3 above)
Comment (dkoshkin) [reply test/scripts/setup-konvoy.sh]: https://wiki.d2iq.com/display/ENG/Download+link
What do you mean its not published, those links would be the S3 link you would use right?

File: test/groups.yaml (hunk 5 above)
Comment (shaneutt) [reply test/groups.yaml]: Yes

File: test/scripts/setup-konvoy.sh (hunk 6 above)
Comment (shaneutt) [reply test/scripts/setup-konvoy.sh]:
```suggestion
KONVOY_VERSION="v1.4.1"
```

File: test/scripts/setup-konvoy.sh (hunk 3 above)
Comment (shaneutt) [reply test/scripts/setup-konvoy.sh]: updated!

**Pull request merged**